### PR TITLE
Add a welcome screen

### LINKED ISSUE #60: Welcome screen in UI when no notebook is selected
When no notebook is selected, we just have a big blank space in the middle. Maybe we should have a little "welcome" screen with getting started instructions.

### PR DESCRIPTION
**Pull request by jeremyrsmith**
Adds a simple welcome page with some instructional garbage text and a list of recent notebooks. It comes up if you don't deep-link into a notebook.

Fixes #60

### CODE REVIEW

**Comment by jeremyrsmith** [comment]
I guess this is also the big unveiling of the terrible logo I made a couple of months ago. We can always change it later 😅

**Comment by indig** [comment]
> I guess this is also the big unveiling of the terrible logo I made a couple of months ago. We can always change it later 😅

You were holding out on me! 😛 Also, I don't see any svg file in the commit, at least not on github.

File: polynote-frontend/polynote/ui.js [hunk 1]
```diff
@@ -1259,8 +1259,38 @@ export class NotebookListUI extends EventTarget {
...
+          <img src="/style/polynote.svg" alt="Polynote" />
+          <h2>Home</h2>
+          
+          <p>
+            To get started, open a notebook by clicking on it in the Notebooks panel, or create a new notebook by
+             clicking the Create Notebook (<span class="create-notebook icon fas"></span>) button.
+          </p>
+          
+          <h3>Recent notebooks</h3>
+          <ul class="recent-notebooks"></ul>
```
Comment (indig) [inline polynote-frontend/polynote/ui.js]: niiiice!! +100

**Comment by indig** [comment]
> > I guess this is also the big unveiling of the terrible logo I made a couple of months ago. We can always change it later 😅
> 
> You were holding out on me! 😛 Also, I don't see any svg file in the commit, at least not on github.

Oh haha never mind, it was in git the whole time I just wasn't looking. Nice :)

**Pull request merged**